test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,21 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull throws Exception {
+        // empty list
+        assertFalse(Utils.isAnyNull(""));
+
+        // only one item
+        assertTrue(Utils.isAnyNull(null));
```

Comment:
Your Travis build fail could be because you didn't cast your null to (Object) null.